client/packages/common/src/ui/icons/Alert.tsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
 
 export const AlertIcon = (
-  props: SvgIconProps & { fill?: string }
+  props: SvgIconProps & { fill?: string; double?: boolean }
 ): JSX.Element => {
-  const { fill = 'none', ...rest } = props;
+  const { fill = 'none', double, ...rest } = props;
```

Comment:
very nice! clever idea to use the prop rather than duplicate the icon file

Reply:
Bonus thought: adding
```
    { icon: <AlertIcon double {...args} />, name: 'Alert (double)' },
```

to the `icon.stories.tsx` file so that the double exclamation is shown in storybook

<img width="344" alt="image" src="https://github.com/user-attachments/assets/e11d65c4-a4f0-407a-924b-576adacac143">

Reply:
Have added this, but I can't get my storybook to run... waiting to validate if that's a me problem or a bigger issue 🤷‍♀️

Reply:
I've tested and it worked, hence the screenshot, so I think it's safe to merge. 🙏